### DISCUSSION TITLE
Question about Expenses Plugin

### DISCUSSION THREAD

**Discussion by vleone64** [kimai/kimai#3715] [General]
I'm just got the Expenses plugin installed, and I have a question about its capability. 

I would like to create a specific expense, such as mileage, that I would have to pay out to employees. I would like them to be able make an expense in Kimai and specify only the quantity. Am I able to make an expense that is able to store a specific cost then let users select that?

What currently happens is every time a new expense is made the cost defaults to zero.

I am running Kimai 1.28.1 with the newest version of the expenses plugin 1.25

**Comment by kevinpapst** [comment]
Give the expense category a cost.

Do test as user, not with your admin account. You see different fields than a regular user.

<img width="914" alt="Bildschirm­foto 2022-12-28 um 18 08 27" src="https://user-images.githubusercontent.com/533162/209847652-b0ffc1c9-ad9b-4066-a3b5-746a6a14dd98.png">

**Comment by vleone64** [comment]
Huh I see. I'll poke around and see if I can find the screen. I see the default Category of "Demo" but I don't see where I can edit it. Thanks.

**Comment by vleone64** [comment]
Found it. Thank you for your help!
